src/libaas/sources.cmake

```diff
@@ -38,6 +38,7 @@ SET(SOURCE_FILES_BASYX
 	${CMAKE_CURRENT_SOURCE_DIR}/basyx/reference.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/basyx/semantics/conceptdescription.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/basyx/semantics/hasdataspecification.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/basyx/serialization/json/deserializer.cpp
```

Comment:
Corresponding header is missing below

Reply:
Overall this really looks good to me, with the both minor changes addressed, this is good to go!